cmake/targets.cmake

```diff
@@ -462,7 +462,7 @@ function(wn_filesystem_files name)
     add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${PARSED_ARGS_OUTPUT_DIR}/${name}.h
                               ${CMAKE_CURRENT_BINARY_DIR}/${PARSED_ARGS_OUTPUT_DIR}/${name}.cpp
       COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/${PARSED_ARGS_OUTPUT_DIR}
-      COMMAND python ${WNFramework_SOURCE_DIR}/utilities/compile_file_directory.py
+      COMMAND ${PYTHON_EXECUTABLE} ${WNFramework_SOURCE_DIR}/utilities/compile_file_directory.py
```

Comment:
Do we need to run `find_package(Python3 COMPONENTS Interpreter REQUIRED)` before trying to use this variable? Might be worth switching to `PYTHON3_EXECUTABLE` since we can't use python2.